modular_skyrat/modules/company_imports/code/armament_datums/bolt_nanotrasen_firearms.dm

```diff
@@ -10,92 +10,103 @@
 /datum/armament_entry/company_import/nanotrasen_bolt_weapons/armor/slim_vest
 	name = "type I vest - slim"
 	item_type = /obj/item/clothing/suit/armor/vest
-	lower_cost = CARGO_CRATE_VALUE
-	upper_cost = CARGO_CRATE_VALUE * 1.5
-	stock_mult = 2
+	cost = PAYCHECK_CREW * 3
```

Comment:
Is it possible for this to cause issues due to the value of the crates they come in?

Reply:
Basically what I said earlier, if they're ordered with the cargo budget they come in valueless large wooden crates that need to be crowbarred. If they are private ordered, they come in a locked briefcase until there are more than five items in which case it'll come in a crate and charge you for the crate as well.